constructs/lambda-service.ts

```diff
@@ -253,9 +256,16 @@ export class LambdaService extends Construct implements iam.IGrantable {
 				);
 			}
 
-			let eventBus = eventBuses[0];
-			if (eventHandler.eventBusName) {
-				const matchedEventBus = eventBuses.find(
+			let eventBus: events.IEventBus;
+			if (!eventHandler.eventBusName) {
+				// Default to first event bus if not specified in handler
+				eventBus = Object.values(eventBuses)[0];
+			} else if (eventBuses[eventHandler.eventBusName]) {
```

Comment:
Defaulting to the first event bus feels fragile. 

While this is nice if you only have one event bus, adding a second event bus later could cause a lot of functions to start publishing to the wrong bus if the new bus is ordered first.

If we want to have a default bus, it would be better to explicitly define the default event bus somewhere.